.coderabbit.yaml

```diff
@@ -1,212 +1,20 @@
-
-        2. When reviewing the documentation files (e.g. `docs/**.md`), they should additionally be linted with help from the tool `markdownlint`, pointing out any issues.
-
-        3. When reviewing the documentation files in `docs/` directory, they should additionally be linted with help from the tool `languagetool`, pointing out any issues.
+  high_level_summary: false
+  review_status: false
+  poem: false
+  collapse_walkthrough: true
+  changed_files_summary: false
   abort_on_close: true
+  path_instructions:
+    - path: "**/*.js"
+      instructions: |
+        Also review the JavaScript code against the following guide: https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/code-review.rst
+    - path: "https://github.com/dimagi/open-source/blob/master/docs/Writing_PRs.md"
+      instructions: |
+        Please follow this guide when evaluating the pull request and make sure that the PR meets all the requirements.
   auto_review:
-    enabled: true
-    auto_incremental_review: true
-    ignore_title_keywords: []
-    labels: []
-    drafts: false
-    base_branches:
-      - stable
-      - master
-      - HOTFIX-*
-  tools:
-    languagetool:
-      enabled: true
-      language: en-US
-      configuration:
-        level: picky
-        mother_tongue: en
-        dictionary:
-          - 'reactive-firewall'
-          - 'CEP-9'
-          - 'CEP-8'
-          - 'CEP-7'
-          - 'CEP-5'
-          - 'Shellscript'
-          - 'bash'
-        disabled_rules:
-          - EN_QUOTES
-          - CONSECUTIVE_SPACES
-        enabled_rules:
-          - STYLE
-          - EN_CONTRACTION_SPELLING
-          - EN_WORD_COHERENCY
-          - IT_IS_OBVIOUS
-          - TWELFTH_OF_NEVER
-          - OXFORD_SPELLING
-          - PASSIVE_VOICE
-    shellcheck:
-      enabled: true
-    ruff:
-      enabled: true
-      configuration:
-        extend_select:
-          - E  # Pycodestyle errors (style issues)
-          - F  # PyFlakes codes (logical errors)
-          - W  # Pycodestyle warnings
-          - N  # PEP 8 naming conventions
-        ignore:
-          - W191
-          - W391
-          - E117
-          - D208
-        line_length: 100
-        dummy_variable_rgx: '^(_.*|junk|extra)$'  # Variables starting with '_' or named 'junk' or 'extras', are considered dummy variables
-        external:
-          flake8-blind-except: {}
-          flake8-docstrings: {}
-          flake8-comprehensions: {}
-          flake8-debugger: {}
-          flake8-eradicate: {}
-          # Include other Flake8 plugins as needed
-    markdownlint:
-      enabled: true
-    yamllint:
-      enabled: true
-      configuration_file: ".yamllint.conf"
-chat:
-  auto_reply: true
+    enabled: false
+    auto_incremental_review: false
+    drafts: true
```

Comment:
I'm not sure we want auto-review on drafts yet. Should we turn it off until we are confident that we want automatic reviews?

Reply:
Sure!